mmdet/integration/nncf/__init__.py

```diff
@@ -1,7 +1,7 @@
 from .compression import (check_nncf_is_enabled, get_nncf_config_from_meta,
                           get_nncf_metadata, get_uncompressed_model,
                           is_checkpoint_nncf, wrap_nncf_model)
-from .compression_hooks import CompressionHook
+from .compression_hooks import CompressionHook, CheckpointHookBeforeTraining
```

Comment:
Please add `CheckpointHookBeforeTraining` to `__all__`.

Reply:
Done